src/modules/storage/Storage.ts

```diff
@@ -0,0 +1,63 @@
+import mkdirp from "mkdirp";
+import path from "path";
+import sqlite3, { Database } from "sqlite3";
+import { IDatabaseConfig } from "../../service/common/Config";
+
+export class Storage {
+    protected database: Database;
+
+    constructor(config: IDatabaseConfig, callback?: (err: Error | null) => void) {
```

Comment:
```suggestion
    constructor(config: IDatabaseConfig, callback: (err: Error | null) => void) {
```
반드시 예외처리를 하도록 설정해주세요